From 611f78ec6a80e1cb985110796f1b12c57ac6091a Mon Sep 17 00:00:00 2001
From: Felix Moessbauer <felix.moessbauer@siemens.com>
Date: Thu, 16 Dec 2021 10:33:05 +0100
Subject: [PATCH 1/1] add support to cross compile packages that depend on this
 package

This patch makes the dependencies of this package more precise,
to enable cross-compilation of packages that depend on python3-gast.

Signed-off-by: Felix Moessbauer <felix.moessbauer@siemens.com>
---
 debian/control | 5 +++--
 debian/rules   | 6 +++++-
 2 files changed, 8 insertions(+), 3 deletions(-)

diff --git a/debian/control b/debian/control
index 2a5e120..ae0d75d 100644
--- a/debian/control
+++ b/debian/control
@@ -6,9 +6,10 @@ Uploaders: Diego M. Rodriguez <diego@moreda.io>
 Build-Depends:
  debhelper-compat (=13),
  dh-python,
- python3-all,
- python3-pytest,
+ libpython3-all-dev,
+ python3-all-dev:any,
  python3-setuptools,
+ python3-pytest <!nocheck>
 Standards-Version: 4.6.0
 Homepage: https://github.com/serge-sans-paille/gast
 Vcs-Git: https://salsa.debian.org/python-team/packages/python-gast.git
diff --git a/debian/rules b/debian/rules
index 6a7a865..d306844 100755
--- a/debian/rules
+++ b/debian/rules
@@ -1,7 +1,11 @@
 #!/usr/bin/make -f
 
 export PYBUILD_NAME=python-gast
-export PYBUILD_TEST_PYTEST=1
+ifneq ($(filter nocheck,$(DEB_BUILD_PROFILES)),)
+	export PYBUILD_TEST_PYTEST=0
+else
+	export PYBUILD_TEST_PYTEST=1
+endif
 
 %:
 	dh $@ --with python3 --buildsystem=pybuild
-- 
2.30.2

